custom/icds_reports/utils/aggregation_helpers/birth_preparedness_forms.py

```diff
@@ -47,7 +47,7 @@ def data_from_ucr_query(self):
         LAST_VALUE(anemia) OVER w as anemia,
         LAST_VALUE(anc_abnormalities) OVER w as anc_abnormalities,
         LAST_VALUE(using_ifa) OVER w as using_ifa,
-        LAST_VALUE(ifa_last_seven_days) OVER w as ifa_last_seven_days,
+        MAX(LAST_VALUE(ifa_last_seven_days) OVER w, 0) as ifa_last_seven_days,
```

Comment:
fixes occurrences where ifa_last_seven_days is negative (email sent to app team)

Reply:
having -ve is valid value or its the worker's mistake

Reply:
We're not sure. Cal has sent an email to Priyanka and others on team to verify if its an app issue or if its valid